app/email/notify-send.js

```diff
@@ -110,29 +108,18 @@ const sendFarmerApplicationEmail = async (data, blob) => {
   }
 
   const emailAddress = data.email
-  let emailTemplateId = templateIdFarmerApplicationGeneration
-  let isSuccess = true
-
-  if (endemics.enabled) {
-    emailTemplateId = data.userType === 'newUser' ? templateIdFarmerApplicationGenerationNewUser : templateIdFarmerApplicationGenerationExistingUser
-  }
+  const emailTemplateId = data.userType === NEW_USER ? templateIdFarmerApplicationGenerationNewUser : templateIdFarmerApplicationGenerationExistingUser
```

Comment:
Now this has finally been changed, we no longer need to plug in the old world template ID, as we only ever set new world new user or new world existing user IDs now. Remove the config for this template ID at this time if ya can please, and we can tidy from the ADO config